components/camel-salesforce/camel-salesforce-component/src/test/java/org/apache/camel/component/salesforce/RestApiIntegrationTest.java

```diff
@@ -571,6 +656,14 @@ public void configure() {
                 from("direct:query").to("salesforce:query?sObjectQuery=SELECT name from Line_Item__c&sObjectClass="
                     + QueryRecordsLine_Item__c.class.getName() + "&format=" + format);
 
+                // testParentRelationshipQuery
+                from("direct:parentRelationshipQuery").to("salesforce:query?sObjectQuery=SELECT LastName, Account.Name FROM Contact WHERE AccountId != null LIMIT 1&sObjectClass="
```

Comment:
Should we query by specific account ID created in the test? This might find a previous record.